pom.xml

```diff
@@ -34,7 +34,7 @@
     <groupId>com.amazon.opendistroforelasticsearch</groupId>
     <artifactId>opendistro_security</artifactId>
     <packaging>jar</packaging>
-    <version>1.3.0.0</version>
+    <version>1.4.0.0</version>
```

Comment:
Since we have released the artifacts for 1.4.0.0, we should be updating this version to `1.5.0.0-SNAPSHOT` which reflects the development for supporting elasticsearch 7.5.0. What do you think ?

Reply:
I agree that we should change the version to 1.5.0.0-SNAPSHOT in the master branch, but I think we should do that after we merge this PR into master and cut a 1.4 branch from it.

Reply:
Okay sounds good. I was confused seeing the artifacts already released. Approved the PR.